client/my-sites/plugins/plugin-details-sidebar/index.jsx

```diff
@@ -53,21 +53,21 @@ const PluginDetailsSidebar = ( {
 
 	const supportLinks = [
 		{
-			href: localizeUrl( 'https://wordpress.com/support/help-support-options' ),
+			supportContext: 'general-support-options',
 			label: translate( 'How to get help!' ),
-			openInNewTab: true,
+			openIn: 'HelpCenter',
```

Comment:
Maybe we could have the `'HelpCenter'` and `'NewTab'` literals as constants to avoid relying on strings to select strategies.